lib/util/storage/database/database_service.dart

```diff
@@ -240,7 +236,7 @@ class DatabaseService {
     }
   }
 
-  Future<WalletStorage> loadWallets() async {
+  Future<dynamic> loadWallets() async {
```

Comment:
I'd use `throw` instead of `return` to be able to keep the type here.
```suggestion
  Future<WalletStorage> loadWallets() async {
```